src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/payment/processor/config/IntegrationContextInitializer.java

```diff
@@ -15,7 +15,7 @@
 import static org.springframework.util.SocketUtils.findAvailableTcpPort;
 
 @Configuration
-@Profile({"integration", "nosb"}) // no servicebus queue handler registration
+@Profile({"integration", "functional"}) // no servicebus queue handler registration
```

Comment:
this class is in `integrationTest` source set

Reply:
And is called `IntegrationContextInitializer`. Do you want this class active in functional tests?